Getting `Module not found: Can't resolve 'fs'` error

I'm using Cypress 10.x inside a Next.js 13.x project with TypeScript.

I getting an error `Module not found: Can't resolve 'fs'` when trying to run the tests. I tried using plugin version 2.0.x and Cypress 11.x and 12.x as well - same result. I tried installing 'fs' inside the project (`npm install --save-dev fs`) but that does not solve the issue either.

The plugin worked without any issues in version 1.7.x with Cypress 9.

Is it only me or is this issue a common occurrence with version 2.1.x?

```bash
> npx cypress run --component --browser electron

warn  - You have enabled experimental feature (scrollRestoration) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://127.0.0.1:8080/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/user/Projects/<project-name>/public' directory

====================================================================================================

  (Run Starting)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Cypress:        12.3.0                                                                         │
  │ Browser:        Electron 106 (headless)                                                        │
  │ Node Version:   v16.19.0 (/opt/homebrew/Cellar/node@16/16.19.0/bin/node)                       │
  │ Specs:          1 found (Button.test.tsx)                                                      │
  │ Searched:       **/*.test.{ts,tsx}                                                             │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                    
  Running:  Button.test.tsx                                                                 (1 of 1)
<i> [webpack-dev-middleware] wait until bundle finished: /__cypress/src/index.html
45 assets
696 modules

ERROR in ./node_modules/cypress-visual-regression/dist/utils.js:1:0
Module not found: Can't resolve 'fs'

Import trace for requested module:
./node_modules/cypress-visual-regression/dist/command.js
./cypress/component/support.ts
../../../Library/Caches/Cypress/12.3.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/@cypress/webpack-dev-server/dist/loader.js!../../../Library/Caches/Cypress/12.3.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/@cypress/webpack-dev-server/dist/browser.js
../../../Library/Caches/Cypress/12.3.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/@cypress/webpack-dev-server/dist/browser.js


https://nextjs.org/docs/messages/module-not-found

client (webpack 5.74.0) compiled with 1 error in 3105 ms

--- a/src/command.js
+++ b/src/command.js
@@ -1,5 +1,4 @@
-const { deserializeError } = require('./utils-browser');
-const { getValueOrDefault } = require('./utils');
+const { deserializeError, getValueOrDefault } = require('./utils-browser');
 
 /* eslint-disable no-undef */
 

--- a/src/plugin.js
+++ b/src/plugin.js
@@ -11,8 +11,8 @@ const {
   createFolder,
   parseImage,
   errorSerialize,
-  getValueOrDefault,
 } = require('./utils');
+const { getValueOrDefault } = require('./utils-browser');
 
 let CYPRESS_SCREENSHOT_DIR;
 

--- a/src/utils-browser.js
+++ b/src/utils-browser.js
@@ -10,4 +10,6 @@ class CompareSnapshotsPluginError extends Error {
 const deserializeError = (error) =>
   new CompareSnapshotsPluginError(JSON.parse(error));
 
-module.exports = { deserializeError };
+const getValueOrDefault = (value, defaultValue) => value || defaultValue;
+
+module.exports = { deserializeError, getValueOrDefault };

--- a/src/utils.js
+++ b/src/utils.js
@@ -79,13 +79,10 @@ const errorSerialize = (error) =>
     )
   );
 
-const getValueOrDefault = (value, defaultValue) => value || defaultValue;
-
 module.exports = {
   adjustCanvas,
   createFolder,
   mkdirp,
   parseImage,
   errorSerialize,
-  getValueOrDefault,
 };

